finders/find_name.py

```diff
@@ -1,7 +1,30 @@
 from typing import List
-
 from finders.stops import Stop
+import re
 
 
 def find_stops_matching_name(stops: List[Stop], name: str, regex: bool = False) -> List[Stop]:
-    pass
+    found = False
+    results = list()
+    if not regex:
+        for element in stops:
+            if element.name == name:
+                print("Your stop is: ", element)
+                found = True
+                return element
+        if not found:
+            print("Sorry, this stop doesn't exist. Try regex=True")
```

Comment:
Same comment about the debug print
Also you should return at least an empty list otherwise the signature would be lying (-> List[Stop])
Or you can change the signature to be -> Optional[List[Stop]] and return None when nothing is found but if find that always returning a least makes more sense